botometerAnalyser/pipelines/retweets.js

```diff
@@ -0,0 +1,89 @@
+const request = require('request-promise');
+
+const usersAnalysis = require('./usersAnalysis');
+const { retweeterIdsQueue } = require('../queues/retweeters');
+const { getTweetQueue } = require('../queues/getTweet');
+
+
+async function analyse({ screenName, tweetId, tweetUrl, responseUrl, requesterUsername }) {
+	getTweetQueue.add({
+		screenName,
+		tweetId,
+		tweetUrl,
+		responseUrl,
+		requesterUsername
+	});
+}
+
+getTweetQueue.on('completed', onGetTweetCompleted);
+getTweetQueue.on('failed', failed);
+
+async function failed(job) {
+	const { tweetUrl, responseUrl, requesterUsername } = job.data;
+
+	return request({
+		url: responseUrl,
+		method: 'POST',
+		json: {
+			text: `@${requesterUsername} I could not found the tweet ${tweetUrl}. Are you sure you spelled it correctly?`,
+			response_type: 'in_channel'
+		},
+	});
+}
+
+
+async function onGetTweetCompleted(job, result) {
+	const {	screenName, responseUrl, tweetId, requesterUsername } = job.data;
+	const tweet = result.data;
+
```

Comment:
Avoid adding a job to get retweets if there are none. Check `tweet.retweet_count`

Reply:
done